Python/samples/cartpole/main.py

```diff
@@ -445,6 +447,9 @@ def CreateSession(
                 sim.episode_start(event.episode_start.config)
                 episode += 1
             elif event.type == "EpisodeStep":
+                # Simulate the next time step using the value of event.episode_step.action.
```

Comment:
instead of time step, I might suggest next "state transition". We can have time based sims, event based sims, sequential problems, etc

Reply:
Good point. Changed.